website/cue/reference/remap/functions/unflatten.cue

```diff
@@ -0,0 +1,95 @@
+package metadata
+
+remap: functions: unflatten: {
+	category: "Enumerate"
+	description: #"""
+		Unflattens the `value` into a nested representation.
+		"""#
+
+	arguments: [
+		{
+			name:        "value"
+			description: "The array or object to unflatten."
+			required:    true
+			type: ["object"]
+		},
+		{
+			name:        "separator"
+			description: "The separator to split flattened keys."
+			required:    false
+			default:     "."
+			type: ["string"]
+		},
+		{
+			name:        "recursive"
+			description: "Whether to recursively unflatten the object values."
+			required:    false
+			default:     "true"
+			type: ["boolean"]
+		},
+	]
+	internal_failure_reasons: []
+	return: types: ["object"]
+
+	examples: [
```

Comment:
Worth adding an example [like this](https://playground.vrl.dev/?state=eyJwcm9ncmFtIjoiLiA9IHVuZmxhdHRlbiguKSIsImV2ZW50Ijp7ImEiOjMsImEuYiI6MiwiYS5jIjo0fSwiaXNfanNvbmwiOmZhbHNlLCJlcnJvciI6bnVsbH0%3D).

Reply:
Thanks! You meant this case, right? https://github.com/vectordotdev/vrl/blob/5f635f93e7bfb060b747339e9a2e965741e0176d/src/stdlib/unflatten.rs#L60
Implemented in https://github.com/vectordotdev/vector/pull/21142/commits/2173ec398db6d095c298585327060421e2aa193e. I was not sure about the example's title

Or did you wanted to depict something like its done here: https://github.com/vectordotdev/vector/blob/beceef4a964a77acc3064c064eb27d265261c17d/website/cue/reference/remap/functions/unnest.cue#L57

Where it's shown that we can replace the original object with `. = unflatten(.)` 
(example of what I mean)
![image](https://github.com/user-attachments/assets/074517b6-6c21-48e6-bd1d-4bb36d765bc8)


I've just noticed that in that `unnest` example, the `input` field that is set on the examples is not being rendered in the website, is this a bug?